extension/src/plots/model/index.ts

```diff
@@ -171,20 +168,20 @@ export class PlotsModel extends ModelWithPersistence {
     }
   }
 
-  public recreateCustomPlots(data?: ExperimentsOutput) {
-    if (data) {
-      this.customCheckpointPlots = collectCustomCheckpointPlots(data)
-    }
-
-    const experiments = this.experiments.getExperiments()
+  public recreateCustomPlots() {
+    const experiments = this.experiments.hasCheckpoints()
+      ? this.experiments
+          .getExperimentsWithCheckpoints()
+          .filter(({ checkpoints }) => !!checkpoints)
+      : this.experiments.getExperiments()
```

Comment:
[B] `collectCustomPlots` should take all experiments regardless of whether or not they have checkpoints. I think the best thing to do here would be call `this.experiments.getExperimentsWithCheckpoints()` all the time as the checkpoints part of `collectCustomPlots` will have a type guard in it anyway?

Reply:
Now that I think about it, there is definitely an easier way to filter out experiment commits. Will update how we filter for now!

Reply:
> Now that I think about it, there is definitely an easier way to filter out experiment commits. Will update how we filter for now!

Originally was thinking I could filter by `type` or `commit` but looks like either of those could be unavailable. Ended up just renaming the variable to make things more clear.